rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -293,10 +296,11 @@ def _to_inst(msg, rostype, roottype, inst=None, stack=[]):
 
 
 def _to_binary_inst(msg):
-    try:
-        return standard_b64decode(msg) if isinstance(msg, str) else bytes(bytearray(msg))
-    except Exception:
+    if isinstance(msg, str):
+        return list(standard_b64decode(msg))
+    if isinstance(msg, list):
```

Comment:
If we accept `list` input like [this](https://github.com/tier4/rosbridge_suite/blob/517e8fa99ac4a61d5676478d0af6a5b3a415c266/test_scripts/int8_array.py#L14), we need this if-condition.